source/jucePlugin/VirusController.cpp

```diff
@@ -476,7 +476,7 @@ namespace Virus
 		case 5:  return "S&G";
 		case 6:  return "Wave3";
 		case 7:  return "Wave4";
-		case 9:  return "Wave5";
```

Comment:
This commit can be squashed to previous instead. The nice art of have a branch / PR

Reply:
i see.  thank you for this info.  immaa git noob... need to study git docs .. sorry for the mess.